src/cli.ts

```diff
@@ -24,12 +24,14 @@ interface CLIOptions {
   paths: Array<string>;
   baseOptions: Options;
   modernizeJS: boolean;
+  output: string;
 }
 
 function parseArguments(args: Array<string>): CLIOptions {
   let paths = [];
   let baseOptions: Options = {};
   let modernizeJS = false;
+  let output = '';
```

Comment:
I'd prefer to keep this as `null` or `undefined` when it has no explicit value, which means something like `let output: string | undefined;`.

Reply:
I’m not sure where that specific error would come from, but it looks like you’re trying to assign a boolean to `output`.